### PR TITLE
Qt-1 Wallet Wizard - UI/UX improvements

### PR DESCRIPTION
**Pull request by dvajdual**
[Qt-1 Wallet Wizard - UI/UX improvements](https://github.com/BlockSettle/terminal/commit/b5bd3c6cbf07fed5bbbd71585be171ba1abd3246)
[Qt-3 Wallet wizard - Success screen]
[QT-9 Wallet wizard - Pages of "Create new wallet" widget with unstandard size show cutted off]

### CODE REVIEW

File: GUI/QtQuick/qml/ConfirmPassword.qml [hunk 1]
```diff
@@ -105,18 +109,18 @@ ColumnLayout  {
...
         }
-
-        onClicked: {
-            click_enter()
-        }
     }
 
     Keys.onEnterPressed: {
-        confirm_but.click_enter()
+         confirm_but.click_enter()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/ConfirmPassword.qml]: indentation is too much, IMO
we normally use 3 spaces, 4 is also good, but 5 is excessive

File: GUI/QtQuick/qml/ConfirmPassword.qml [hunk 2]
```diff
@@ -105,18 +109,18 @@ ColumnLayout  {
...
     }
 
     Keys.onReturnPressed: {
-        confirm_but.click_enter()
+         confirm_but.click_enter()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/ConfirmPassword.qml]: indentation

File: GUI/QtQuick/qml/StyledControls/CustomButton.qml [hunk 3]
```diff
@@ -37,6 +39,22 @@ Button {
...
 
         radius: 14
+
+        border.color: preferred ? BSStyle.buttonsPreferredBorderColor : BSStyle.buttonsStandardBorderColor
+        border.width: control.activeFocus? 1 : 0
+
+    }
+
+    Keys.onEnterPressed: {
+         click_enter()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/StyledControls/CustomButton.qml]: indentation (and below)

File: GUI/QtQuick/qml/SuccessNewWallet.qml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        Component.onCompleted: {
+            finish_but.preferred = true
+        }
+        function click_enter() {
+            sig_finish()
+        }
+    }
+
+    Keys.onEnterPressed: {
+         finish_but.click_enter()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/SuccessNewWallet.qml]: indentation (and below)

File: GUI/QtQuick/qml/TermsAndConditions.qml [hunk 5]
```diff
@@ -108,5 +108,14 @@ ColumnLayout  {
         Layout.fillWidth: true
         height: 24
     }
+
+
+    Keys.onEnterPressed: {
+         continue_but.click_enter()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/TermsAndConditions.qml]: indentation (and below)

File: GUI/QtQuick/qml/ConfirmPassword.qml (hunk 1 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/ConfirmPassword.qml]: Ok agree

Comment (dvajdual) [reply GUI/QtQuick/qml/ConfirmPassword.qml]: default - 4 spaces...strange that I have more. I will do 4

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**